src/source/Rtcp/RollingBuffer.c

```diff
@@ -137,7 +137,11 @@ STATUS rollingBufferExtractData(PRollingBuffer pRollingBuffer, UINT64 index, PUI
     isLocked = TRUE;
     if (pRollingBuffer->headIndex > index && pRollingBuffer->tailIndex <= index) {
         *pData = pRollingBuffer->dataBuffer[rollingBufferMapIndex(pRollingBuffer, index)];
```

Comment:
Store the value of rollingBufferMapIndex(pRollingBuffer, index) in a local variable to be used here and two lines below